### PR TITLE
Add Top Issues to Probot App directory

### PR DESCRIPTION
**Pull request by adamzolyak**
##### Checklist

- [x] If you're adding a listing for your app, be sure your app is in the `/_apps/` folder before opening this Pull Request.
- [x] All comments in frontmatter need to be removed.
- [x] Performs a useful action through the GitHub API that solves an existing problem for developers: {makes it easier for contributors to identify top issues that would benefit the most community members}
- [x] Is original: for example, it does something not already done by an existing Probot app
- [x] [Has tests](https://github.com/adamzolyak/gh-vote-bot/blob/master/index.test.js)
- [x] [Has documentation](https://github.com/adamzolyak/gh-vote-bot/blob/master/README.md)
- [x] [Is open source](https://github.com/adamzolyak/gh-vote-bot/blob/master/LICENSE)
- [x] [Has a license](https://github.com/adamzolyak/gh-vote-bot/blob/master/LICENSE)
- [x] [Has a code of conduct](https://github.com/adamzolyak/gh-vote-bot/blob/master/CODE_OF_CONDUCT.md)
- [x] Has someone willing to at least minimally maintain them for the near future: {adamzolyak}

-----
[View rendered _apps/topissues.md](https://github.com/adamzolyak/probot.github.io/blob/patch-1/_apps/topissues.md)

### CODE REVIEW

**Comment by adamzolyak** [comment]
The remaining lint failures are not for my app.  If you'd like me to take a look at fixing them, let me know.

File: _apps/topissues.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+# A human-friendly name of your listing
+title: Top Issues
+
+# A short description of what your app does
+description: Labels issues with the most "+1" emoji reactions
+
+# The slug of your hosted app on GitHub (https://github.com/apps/YOUR-SLUG)
+slug: https://github.com/apps/top-issue
```
Comment (hiimbex) [inline _apps/topissues.md]: This should be just `top-issue`, but it doesn't appear that you actually own the slug given https://github.com/apps/top-issue 404s

File: _apps/topissues.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- https://raw.githubusercontent.com/adamzolyak/gh-vote-bot/master/docs/issue_detail.png
+
+# The GitHub usernames of anyone who authored the app
+authors: [adamzolyak]
+
+# The repository where the code is located
+repository: adamzolyak/gh-vote-bot
+
+# The address where this app is deployed
+host: https://gh-vote-bot.herokuapp.com/
```
Comment (hiimbex) [inline _apps/topissues.md]: This will need to be: `https://gh-vote-bot.herokuapp.com`

File: _apps/topissues.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+---
+
+## How It Work
```
Comment (hiimbex) [inline _apps/topissues.md]: Maybe: `How It Works`

File: _apps/topissues.md (hunk 1 above)
Comment (adamzolyak) [reply _apps/topissues.md]: `top-issues`.  updated.  

https://github.com/apps/top-issues-bot

**Comment by adamzolyak** [comment]
My keep alive wasn't keeping Heroku running.  Testing now.  Will post an update once I've confirmed the keep alive is working as expected.

Also I noticed the note about `Heads up! Note that most apps in the @probot organization have an official hosted app that you can use for your open source project. Use the hosted instance if you don't want to deploy your own.` at https://probot.github.io/docs/deployment/.  How's that work - can I move my app into the probot org?

**Comment by adamzolyak** [comment]
The keepalive is now working as expected.  You can run a test.  Runs every 25 minutes.

**Pull request merged**